### DISCUSSION TITLE
1.1.12 Nuget release

### DISCUSSION THREAD

**Discussion by MattB-msft** [microsoft/PowerPlatform-DataverseServiceClient#388] [General]
## What's Changed
* Push Update from Main Codebase by @MattB-msft in https://github.com/microsoft/PowerPlatform-DataverseServiceClient/pull/387
Refactored CurrentAccessToken property to avoid async call.
Fix for min version of default client, pre-connection so it is recognized by features that require 9.0 or greater. Fixes #381 
Fix for notification of unsupported features when not 'breaking' to alert as 'warnings' in log vs information.
Dependency changes:
   Microsoft.Rest.Client moved to 2.3.24 due to CVE-2022-26907.

**Full Changelog**: https://github.com/microsoft/PowerPlatform-DataverseServiceClient/compare/1.1.9...1.1.12

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/PowerPlatform-DataverseServiceClient/releases/tag/1.1.12'>1.1.12 Nuget release</a>.</em>

**Comment by Franetse7** [comment]
Salut la communauté
